AlphaWallet/TokenScriptClient/Models/AssetDefinitionStore.swift

```diff
@@ -35,7 +35,6 @@ class AssetDefinitionStore: NSObject {
     private var lastContractInPasteboard: String?
     private var backingStore: AssetDefinitionBackingStore
 
-    lazy var assetAttributesCache: AssetAttributesCache = AssetAttributesCache()
```

Comment:
Yes, let's remove it since it's not used.

But is writing to the cache the reason why scrolling is jerky? I just tried on 4e4478cbc7a8baa5d948d5998bff150ca7b84f2b and it might not be a problem any more. But lets  remove it any way.